common/scala/src/main/scala/whisk/http/BasicHttpService.scala

```diff
@@ -86,39 +86,57 @@ trait BasicHttpService extends HttpService with TransactionCounter {
      */
     def loglevelForRoute(route: String): Logging.LogLevel = Logging.InfoLevel
 
+    /** Rejection handler to terminate connection on a bad request. Delegates to Akka handler. */
+    val prioritizeRejections = recoverRejections { rejections =>
+        val priorityRejection = rejections.find {
+            case rejection: UnacceptedResponseContentTypeRejection => true
+            case _ => false
+        }
+
+        priorityRejection.map(rejection => Rejected(Seq(rejection))).getOrElse(Rejected(rejections))
+    }
+
+    val toStrict = mapInnerRoute { innerRoute =>
```

Comment:
In general what is the background of this conversion? What's the difference between this and `toStrictEntity`?